tests/rootfs_boot.py

```diff
@@ -146,6 +146,12 @@ def boot(self, reflash=True):
             if wan:
                 wan.sendline('\nip -6 route del default')
                 wan.expect(prompt)
+                # Add eth1 ipv6 default gateway
+                default_gateway_ipv6 = self.config.board['common']['Default_gateway_ipv6']
```

Comment:
this failed on the unit test, maybe if 'common' exists add the route?

Reply:
Doesn't ipv6 routes get set by router advertisements? Why isn't your environment setup properly.

Can you review @ketantewari